src/pkg/control/options.go

```diff
@@ -48,44 +47,50 @@ func Defaults() Options {
 }
 
 // ---------------------------------------------------------------------------
-// Restore Item Collision Policy
+// Restore Configuration
 // ---------------------------------------------------------------------------
 
-// CollisionPolicy describes how the datalayer behaves in case of a collision.
-type CollisionPolicy int
-
-//go:generate stringer -type=CollisionPolicy
 const (
-	Unknown CollisionPolicy = iota
-	Copy
-	Skip
-	Replace
+	defaultRestoreLocation = "Corso_Restore_"
 )
 
-// ---------------------------------------------------------------------------
-// Restore Destination
-// ---------------------------------------------------------------------------
+// CollisionPolicy describes how the datalayer behaves in case of a collision.
+type CollisionPolicy string
 
 const (
-	defaultRestoreLocation = "Corso_Restore_"
+	Unknown CollisionPolicy = ""
+	Skip    CollisionPolicy = "skip"
+	Copy    CollisionPolicy = "copy"
+	Replace CollisionPolicy = "replace"
 )
 
-// RestoreDestination is a POD that contains an override of the resource owner
-// to restore data under and the name of the root of the restored container
-// hierarchy.
-type RestoreDestination struct {
-	// ResourceOwnerOverride overrides the default resource owner to restore to.
-	// If it is not populated items should be restored under the previous resource
-	// owner of the item.
-	ResourceOwnerOverride string
-	// ContainerName is the name of the root of the restored container hierarchy.
-	// This field must be populated for a restore.
-	ContainerName string
+// RestoreConfig contains
+type RestoreConfig struct {
+	// Defines the per-item collision handling policy.
+	// Defaults to Skip.
+	OnCollision CollisionPolicy
+
+	// ProtectedResource specifies which resource the data will be restored to.
+	// If empty, restores to the same resource that was backed up.
+	// Defaults to empty.
+	ProtectedResource string
+
+	// Location specifies the container into which the data will be restored.
+	// Only accepts container names, does not accept IDs.
+	// If empty or "/", data will get restored in place, beginning at the root.
+	// Defaults to "Corso_Restore_<current_dttm>"
+	Location string
+
+	// Drive specifies the drive into which the data will be restored.
+	// If empty, data is restored to the same drive that was backed up.
+	// Defaults to empty.
+	Drive string
 }
 
-func DefaultRestoreDestination(timeFormat dttm.TimeFormat) RestoreDestination {
-	return RestoreDestination{
-		ContainerName: defaultRestoreLocation + dttm.FormatNow(timeFormat),
+func DefaultRestoreConfig(timeFormat dttm.TimeFormat) RestoreConfig {
+	return RestoreConfig{
+		OnCollision: Skip,
```

Comment:
is this a safe default? It's different than the behavior we have currently. What we have now is closes to copy I think

Reply:
We don't technically have a default, since we always create a new folder, and therefore there can be no collisions.  But, yes, this an appropriately safe default.